src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -105,12 +98,30 @@ public static void setDashboards(String user, Set<String> projects, Set<String>
 		else {
 			List<String> common = new ArrayList<String>(indexPatterns);
 			
+			// build normalizedSort
+			List<String> normalizedSort = new ArrayList<String>();
```

Comment:
Don't I need to then cast everything later on, such as with line 117?  Not sure if we gain anything by doing this...